test/e2e/engine-vllm-pvc/test.sh

```diff
@@ -38,3 +38,5 @@ for i in {1..10}; do
     -H "Content-Type: application/json" \
     -d '{"model": "opt-125m-cpu", "prompt": "Who was the first president of the United States?", "max_tokens": 40}'
 done
+
+helm uninstall kubeai-models # cleans up above model helm chart on success
```

Comment:
This is probably an oversight from before, we should be doing a `kubectl delete models --all` in the wrapper after each test case.

Reply:
We will want to remove the helm installation, not just the models, maybe in the wrapper?

Reply:
That should happen (I wonder if it is not removing the CRDs, or not executing):

```bash
cleanup() {
    echo "Running test framework cleanup..."
    if [ $skaffold_pid != 9999999999999999 ]; then
        kill $skaffold_pid
        wait $skaffold_pid
    fi
    skaffold delete -f $REPO_DIR/skaffold.yaml
}
```

Reply:
It cleans up the helm chart for kubai, just not kubai models

Reply:
There is some risk that we may actually remove someone's production models by running `helm uninstall kubeai-models`. Let's just remove this line for now. I generally do `kind delete cluster` to clean up between e2e test runs.